extensions/amp-ad-exit/0.1/config.js

```diff
@@ -121,6 +134,21 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
+      const vendor = target.vars[variable]['vendorAnalyticsSource'];
+      if (vendor !== undefined) {
```

Comment:
Why not just if (vendor)?  I don't see why you'd be interested in null/undefined/empty string (and presumably this could never be the number 0).

Reply:
is that true? Shouldn't the name always be a non-empty string?

Reply:
Good point. Will fix.